### PR TITLE
show data in service view

### PR DESCRIPTION
**Pull request by YitongFeng**
image

### CODE REVIEW

File: webportal/src/app/cluster-view/services/service-table.component.ejs [hunk 1]
```diff
@@ -8,12 +8,24 @@
...
+    <% for(let i = 0; i < data.length; i ++) { %>
       <tr>
-        <td> </td>
-        <td> </td>
-        <td> </td>
-        <td> </td>
+        <td><%= data[i].node.metadata.name %></td>
+        <td><%= data[i].node.metadata.name %></td>
+        <td>
+          <% for(var k in data[i].node.metadata.labels) { %>
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/service-table.component.ejs]: `let k`

File: webportal/src/app/cluster-view/services/service-view.component.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.btn{
+	padding: 3px 3px;
+	margin: 2px;
+	font-size: 12px
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/service-view.component.scss]: 2 space indent, semicolon here

File: webportal/src/app/cluster-view/services/services.component.js [hunk 3]
```diff
@@ -35,24 +36,13 @@ const serviceViewHtml = serviceViewComponent({
...
-        $('#service-table').html(serviceTableComponent({
-          services: data      
-        }));
-      }
-    },
-    error: (xhr, textStatus, error) => {
-      const res = JSON.parse(xhr.responseText);
-      alert(res.message);
-    }
+  service.getServiceView("http://10.151.40.222:8080", "default", (data) => {
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/services.component.js]: do not hard code uri, use `webportalConfig` instead

File: webportal/src/app/cluster-view/services/service-table.component.ejs (hunk 1 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/service-table.component.ejs]: solved

File: webportal/src/app/cluster-view/services/service-view.component.scss (hunk 2 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/service-view.component.scss]: solved

File: webportal/src/app/cluster-view/services/services.component.js (hunk 3 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/services.component.js]: Solved in 4th commit.

File: webportal/src/app/cluster-view/services/services.component.js [hunk 4]
```diff
@@ -35,24 +36,13 @@ const serviceViewHtml = serviceViewComponent({
...
-        $('#service-table').html(serviceTableComponent({
-          services: data      
-        }));
-      }
-    },
-    error: (xhr, textStatus, error) => {
-      const res = JSON.parse(xhr.responseText);
-      alert(res.message);
-    }
+  service.getServiceView('${webportalConfig.k8sApiServerUri}', 'default', (data) => {
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/services.component.js]: you need to uncomment `require webportal.config.json` line first to use `webportalConfig`.
`'${webportalConfig.k8sApiServerUri}'` doesn't work, use variable `webportalConfig.k8sApiServerUri` straightly.

File: webportal/src/app/cluster-view/services/service-info.js [hunk 5]
```diff
@@ -57,3 +57,5 @@ const getNodePods = (kubeURL, namespace, nodeList, callback) => {
         }
     });
```
Comment (abuccts) [inline webportal/src/app/cluster-view/services/service-info.js]: help to refine style in `service-info.js`?

Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/service-info.js]: I added a line module.exports = { getServiceView } at the end.

File: webportal/src/app/cluster-view/services/services.component.js (hunk 4 above)
Comment (YitongFeng) [reply webportal/src/app/cluster-view/services/services.component.js]: Resolved. Thanks

File: webportal/src/app/cluster-view/services/service-info.js (hunk 5 above)
Comment (abuccts) [reply webportal/src/app/cluster-view/services/service-info.js]: I mean could you pls help to refine other styles in this file, like indent, semicolon, etc,

**Pull request merged**